txpipe/theory.py

```diff
@@ -0,0 +1,287 @@
+from .base_stage import PipelineStage
+from .data_types import FiducialCosmology, SACCFile
+import numpy as np
+
+
+class TXTwoPointTheoryReal(PipelineStage):
```

Comment:
Is it useful to consolidate the two classes in real and fourier? there's a lot of repetition it seems?

Reply:
nvm, seems different enough it might be too much trouble